src/Confluent.Kafka/AdminClient.cs

```diff
@@ -603,6 +644,19 @@ public Task DeleteTopicsAsync(IEnumerable<string> topics, DeleteTopicsOptions op
             return completionSource.Task;
         }
 
+        /// <summary>
+        ///     Refer to <see cref="Confluent.Kafka.IAdminClient.DeleteGroupAsync(IList{string}, DeleteGroupOptions)" />
+        /// </summary>
+        public Task DeleteGroupAsync(IList<string> groups, DeleteGroupOptions options = null)
```

Comment:
`DeleteConsumerGroupsAsync`

Reply:
Change to plural but avoid adding consumer - see comment above.